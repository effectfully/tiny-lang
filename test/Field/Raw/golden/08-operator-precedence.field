ext a;
ext b;
ext c;

assert
    a + b [c] == a + (b [c])
and a + b [c] <= (a + b) [c]
;
