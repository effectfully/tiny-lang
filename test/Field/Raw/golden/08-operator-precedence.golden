Program [Var "a",Var "b",Var "c"] Statements [EAssert (EAppBinOp And (EAppBinOp FEq (EAppBinOp Add (EVar (Var "a")) (EAppBinOp BAt (EVar (Var "c")) (EVar (Var "b")))) (EAppBinOp Add (EVar (Var "a")) (EAppBinOp BAt (EVar (Var "c")) (EVar (Var "b"))))) (EAppBinOp FLe (EAppBinOp Add (EVar (Var "a")) (EAppBinOp BAt (EVar (Var "c")) (EVar (Var "b")))) (EAppBinOp BAt (EVar (Var "c")) (EAppBinOp Add (EVar (Var "a")) (EVar (Var "b"))))))]